### PR TITLE
[ABW-3915] Expiration errors

### PR DESCRIPTION
**Pull request by matiasbzurovski**
Jira ticket: [ABW-3915](https://radixdlt.atlassian.net/browse/ABW-3915)

## Changelog
- Updates `DappToWalletInteractionSubintentExpireAtTime` to handle value as `u64`.
- Add logic to check the `ExpirationStatus` at a given time.

[ABW-3915]: https://radixdlt.atlassian.net/browse/ABW-3915?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apple/Sources/Sargon/Extensions/Methods/RadixConnect/WalletInteraction/DappToWalletInteractionSubintentExpiration+Wrap+Functions.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Foundation
+import SargonUniFFI
+
+extension DappToWalletInteractionSubintentExpiration {
+	public func getStatus() -> DappToWalletInteractionSubintentExpirationStatus {
```
Comment (matiasbzurovski) [inline apple/Sources/Sargon/Extensions/Methods/RadixConnect/WalletInteraction/DappToWalletInteractionSubintentExpiration+Wrap+Functions.swift:5]: Can you please Kotlin analogue method @micbakos-rdx ?

File: apple/Sources/Sargon/Extensions/Methods/RadixConnect/WalletInteraction/DappToWalletInteractionSubintentExpireAtTime+Wrap+Functions.swift [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Foundation
+import SargonUniFFI
+
+extension DappToWalletInteractionSubintentExpireAtTime {
+	public var date: Date {
+		.init(timeIntervalSince1970: TimeInterval(unixTimestampSeconds))
```
Comment (matiasbzurovski) [inline apple/Sources/Sargon/Extensions/Methods/RadixConnect/WalletInteraction/DappToWalletInteractionSubintentExpireAtTime+Wrap+Functions.swift:6]: It was helpful for iOS, not sure if you need it in Android as well @micbakos-rdx

**Review by CyonAlexRDX — APPROVED** [review]
LGTM

**Comment by micbakos-rdx** [comment]
@matiasbzurovski I pushed the extension

**Pull request merged**